changelog/upgrade-notes.md

```diff
@@ -4,6 +4,11 @@
 
 #### Backwards incompatible changes
 
+##### Allow declutter in image render mode
```

Comment:
This is only a behavioral change, not a breaking change.

Reply:
OK, I moved the text to the "other" section.